selinux/tendrl.te

```diff
@@ -0,0 +1,76 @@
+policy_module(tendrl, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type tendrl_t;
+type tendrl_exec_t;
+init_daemon_domain(tendrl_t, tendrl_exec_t)
+
+type tendrl_conf_t;
+files_config_file(tendrl_conf_t)
+
+type tendrl_log_t;
+logging_log_file(tendrl_log_t)
+
+type tendrl_var_lib_t;
+files_type(tendrl_var_lib_t)
+
+type tendrl_var_run_t;
+files_pid_file(tendrl_var_run_t)
+
+type tendrl_unit_file_t;
+systemd_unit_file(tendrl_unit_file_t)
+
+type collectd_conf_t;
+files_config_file(collectd_conf_t)
+
+permissive tendrl_t;
+
+########################################
+#
+# tendrl local policy
+#
+allow tendrl_t self:capability { sys_rawio sys_admin net_admin };
+allow tendrl_t self:capability2 block_suspend;
+allow tendrl_t self:fifo_file rw_fifo_file_perms;
+allow tendrl_t self:unix_stream_socket create_stream_socket_perms;
+allow tendrl_t self:tcp_socket { accept listen };
+allow tendrl_t self:netlink_route_socket nlmsg_write;
+
+allow tendrl_t tendrl_conf_t:dir manage_dir_perms;
+allow tendrl_t tendrl_conf_t:file manage_file_perms;
+
+corecmd_exec_shell(tendrl_t)
+corecmd_exec_bin(tendrl_t)
+
+corenet_tcp_bind_generic_node(tendrl_t)
+corenet_tcp_bind_http_cache_port(tendrl_t)
+corenet_tcp_connect_amanda_port(tendrl_t)
+
+optional_policy(`
+	collectd_systemctl(tendrl_t)
+')
+
+optional_policy(`
+	tendrl_stub_collectd()
+	allow collectd_t self:capability setgid;
+	tendrl_domtrans(collectd_t)
+	allow collectd_t self:netlink_audit_socket { nlmsg_relay create };
+	sudo_exec(collectd_t)
+	sysnet_exec_ifconfig(collectd_t)
+	corecmd_exec_bin(collectd_t)
+	corecmd_exec_shell(collectd_t)
+	allow tendrl_t collectd_conf_t:dir manage_dir_perms;
+	allow tendrl_t collectd_conf_t:file manage_file_perms;
+	allow collectd_t collectd_conf_t:file manage_file_perms;
+	allow collectd_t collectd_conf_t:dir manage_dir_perms;
+	allow collectd_t self:capability { setuid chown sys_resource audit_write };
+	allow collectd_t self:unix_stream_socket { connectto };
+	allow dmidecode_t collectd_t:fifo_file { getattr open read };
```

Comment:
This twho rules should be fixed in dmidecote SELinux security module

Reply:
Sure, i will remove it from here!